package.json

```diff
@@ -19,32 +19,18 @@
     "url": "https://github.com/webrtc/samples.git"
   },
   "scripts": {
-    "test": "gulp",
-    "start": "http-server . -c-1"
+    "start": "http-server . -c-1",
+    "test": "npm run eslint && npm run stylelint",
+    "eslint": "eslint 'src/content/**/*.js'",
```

Comment:
note the single quotes here. This is cross-os glob matching, not shell glob matching